tiledb/bioimg/helpers.py

```diff
@@ -178,9 +187,13 @@ def iter_color(attr_type: np.dtype, channel_num: int = 3) -> Iterator[Dict[str,
             )
             blue = np.random.randint(low=min_val, high=max_val, dtype=attr_type).item(0)
         else:
-            red = np.random.uniform(low=min_val, high=max_val).astype(attr_type)
-            green = np.random.uniform(low=min_val, high=max_val).astype(attr_type)
-            blue = np.random.uniform(low=min_val, high=max_val).astype(attr_type)
+            raise NotImplementedError(
+                "RGB-Float and RGB-1-0 formats are not yet supported for this iterator"
+            )
+            # get_decimal_from_rgba is not support for floats
```

Comment:
Dead code?

Reply:
SGTM, can retrieve from git history if needed. Please go ahead and merge after the update.